src/applications/vaos/tests/appointment-list/components/ConfirmedAppointmentDetailsPage/index.video.unit.spec.jsx

```diff
@@ -807,87 +806,35 @@ describe('VAOS <ConfirmedAppointmentDetailsPage>', () => {
         .be.ok;
     });
 
-    it('should fire a print request when print button clicked', async () => {
+    it('should print appointment details', async () => {
```

Comment:
I opted to simplify this test, since it doesn't benefit from going from the list to the details page.

Also, in the future, we could probably check the print link in a `// Then` section of another details page test.